resources/public/api/conf/2.0/application.raml

```diff
@@ -616,4 +616,53 @@ traits:
                         description: The specified arrival does not exist, has been archived or is not available to the EORI number.
                         value:
                           code: NOT_FOUND
+        /{messageId}:
+          get:
+            displayName: Get a cached message related to a arrival and message ID
+            description: |
+              Retrieve all messages related to a specific arrival and message ID that have been received within the past 31 days.
 
+              For example, this could include messages about route diversions or unloading permissions.
+
+              Movements older than 31 days and their related messages are archived and cannot be returned by the API.
+
+               The response can be returned in one of the following formats based on the provided `Accept` header:
+
+               * `application/vnd.hmrc.2.0+json-xml`: JSON-wrapped XML, where the message is provided as an XML string sent as part of JSON payload containing other metadata.
+
+               **Note:** You can view a transit movement only if your Economic Operators Registration and Identification (EORI) number matches the EORI number that created the transit declaration for that movement. Your EORI number is part of the bearer token that you use for API authentication.
+            is:
+              - acceptSingleMessageGetHeader
+              - acceptHeaderInvalid
+            (annotations.scope): "common-transit-convention-traders"
+            securedBy: [ sec.oauth_2_0: { scopes: [ "common-transit-convention-traders" ] } ]
+            responses:
+              200:
+                body:
+                  application/json+xml:
```

Comment:
Just going to check this works using the PREVIEW_RAML service manager profile.

Reply:
It works... and I'm a fool because I'm the one who did this in departures (and completely forgot). Sorry!

Reply:
Just to clarify why I did this in departures, it's because I wanted two different responses to appear on the definition page, and this was the only way I could do it. So it's fine -- but I wonder how this'll translate into OAS.